src/modules/popUpConsultation.js

```diff
@@ -0,0 +1,23 @@
+import showModal from './showModal';
+import Validate from './validate';
+
+const popUpConsultation = result => {
+  const directorForm = document.querySelector('.director-form');
+  const popUpConsultation = document.querySelector('.popup-consultation');
+  const consultQuestion = document.querySelector('.consult-question');
+
+  const isValid = new Validate(directorForm);
```

Comment:
Classes should be named as nouns

```suggestion
  const isValid = new FormValidator(directorForm);
```

The same for variable names:

```suggestion
  const formValidator = new FormValidator(directorForm);
```